src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -398,4 +411,89 @@ class TableView extends React.Component<TableViewProps> {
   }
 }
 
+const UNSEARCHABLE_FIELDS = ['p99', 'p75', 'p95', 'last_seen'];
+
+const ExpandAggregateRow = (props: {
+  children: React.ReactNode;
+  eventView: EventView;
+  column: TableColumn<keyof TableDataRow>;
+  dataRow: TableDataRow;
+  location: Location;
+}) => {
+  const {children, column, dataRow, eventView, location} = props;
+
+  const {eventViewField} = column;
+
+  const exploded = explodeField(eventViewField);
+  const {aggregation} = exploded;
+
+  if (aggregation === 'count') {
+    return (
+      <Tooltip title={t('Expand aggregated row')}>
+        <a
```

Comment:
Should we use `button` elements for cases like this? You wouldn't have to put in bogus href attributes then.

Reply:
What about a button element?

Reply:
@markstory Thinking about this more, I'm going to use `<Link />` instead to create genuine URLs: https://github.com/getsentry/sentry/pull/15793/commits/ef684d48d4111529738b08a56873ba7d34cf15c5

Reply:
This is primary to be able to open links in new tabs.